MIGRATION_GUIDE.md

```diff
@@ -1,18 +1,18 @@
 # Migration Guide
 
-The [v1 release](https://github.com/pedronauck/docz/pull/656) was one of our big releases and a lot of breaking changes was introduced. So, there's few apis that changed and you need to update your code if you're coming from previous versions. It's not a big deal, but you need to follow this guide in order to get Docz running properly on your project.
+The [v1 release](https://github.com/pedronauck/docz/pull/656) was one of our big releases so far and a lot of breaking changes was introduced. So, there's a few apis that changed and you need to update your code if you're coming from previous versions. It's not a big deal, but you need to follow this guide in order to get Docz running properly on your project after an upgrade.
 
 ## Update React to use Hooks
 
 We made a [huge improvement](https://github.com/pedronauck/docz/commit/f57f987df0536b3b65a26f1b0e8a8f8f00d63800) on docz using the new react hooks. So, the biggest requirement is that you need `react` and `react-dom` with the version `>= 16.8.0`, because that's the version that has hooks released and stable. So, just update your React version, it's fully retro compatible.
 
 ## Spectrum instead of Discord
 
-Another thing that we changed, it's now we're attending on Spectrum instead of Discord. You can check the [Docz community](https://spectrum.chat/docz) and ask we whatever you want!
+Another thing that we changed, it's now we're attending on Spectrum instead of Discord. You can check the [Docz community](https://spectrum.chat/docz) and ask us whatever you want!
```

Comment:
```suggestion
Another thing that has changed is our community is now on Spectrum instead of Discord. You can check the [Docz community](https://spectrum.chat/docz) and ask us whatever you want!
```